.github/workflows/release-please.yml

```diff
@@ -173,6 +173,8 @@ jobs:
     with:
       tag: ${{ needs.release-please.outputs.tag }}
     secrets: inherit
+    environment:
```

Comment:
This is good..these jobs specifically pushes the binaries, images, docs...`release-please` job will actually create the release so you will probably need to add environment there as well